recipes/orange3/meta.yaml

```diff
@@ -9,8 +9,6 @@ source:
 
 build:
   number: 0
-  script:
-    - python setup.py install
```

Comment:
@astaric you need to use: 

`script: python setup.py install --single-version-externally-managed --record record.txt`

PS: You need the same `setuptools` option from your `build.sh` and `bld.bat` scripts to avoid having to install all the run dependencies at build time (` --single-version-externally-managed`).